requirements_spark.txt

```diff
@@ -8,7 +7,18 @@ numpy==1.19.4
 git+https://github.com/metabrainz/brainzutils-python.git@v1.14.1
 pytest == 6.2.1
 pytest-cov == 2.10.1
-py4j == 0.10.9.1
+py4j == 0.10.7
 pyspark == 2.4.5
 pydantic == 1.6.1
 unidecode == 1.1.2
+Flask==1.1.2
+werkzeug==1.0.1
+Flask-DebugToolbar==0.11.0
+Flask-UUID==0.2
+raven[flask]==6.10.0
+redis==3.5.3
+msgpack-python==0.5.6
+requests==2.23.0
+SQLAlchemy==1.3.16
+mbdata==25.0.4
+sqlalchemy-dst==1.0.1
```

Comment:
Double-check these values, the spark setup doesn't run a flask webserver, so most of these are probably not needed

Reply:
spark does use the manage.py cli, which means that we do need both BU, Flask, and all of the related dependencies. This is fine to leave here